cache-filter/src/log.rs

```diff
@@ -0,0 +1,55 @@
+use proxy_wasm::types::LogLevel;
+
+#[macro_export(local_inner_macros)]
+macro_rules! log {
+    (context: $context:expr, $lvl:expr, $($arg:tt)+) => ({
```

Comment:
context_id is required for combing logs with the identical context or else logs from different contexts will be there.

Reply:
How will I get access to context_id without it?

Reply:
I think you misunderstood what I mean: I mean to replace `(context: $context:expr, ...)` with `($context:expr, ...)` so that the callers don't need to call this with something like `log!(context: self.context_id, ...)` but instead with `log!(self.context_id, ...)`.

Reply:
Oh! will add this into the next commit.